docs/EiffelVis/content/_index/features-2/graphoptions.md

```diff
@@ -0,0 +1,17 @@
++++
+fragment = "item"
+#disabled = false
+date = "2022-01-05"
+weight = 115
+background = "dark"
+align = "top"
+
+title = "Graph Options"
```

Comment:
And here like Karl already said in slack, "Modify how you want to display your graph"